### ISSUE TITLE
Global table upload is only working sporadically

### ISSUE THREAD

**Issue by audiodude** [openzim/wp1#249]
https://en.wikipedia.org/w/index.php?title=User:WP_1.0_bot/Tables/OverallArticles&action=history

I looked at the log file inside the workers container (`/var/log/enqueue-global.cron.log`) and it doesn't contain any errors, and was updated today, so it's a mystery what's going on.

**Comment by audiodude** [comment]
Also didn't see any errors in the FAILURE bucket in rq.

**Comment by audiodude** [comment]
I just ran `enqueue-global.py` manually and monitored the queue. According to the logs, it processed and uploaded the global table successfully. However, I don't see any updates on Wikipedia.

Currently my best guess is that the table data is maybe being cached or otherwise duplicated, and the wikicode for the "new" table is identical to the old so is getting rejected by Wikipedia.

**Comment by audiodude** [comment]
I ran the Global Table generation code again, locally, with it printing out the wikicode and I verified that the wikicode it generated is identical to the current revision on the table (https://en.wikipedia.org/w/index.php?title=User:WP_1.0_bot/Tables/OverallArticles&oldid=984705414).

That's a start, now to figure out why the global numbers haven't changed in nearly 10 days. It seems extremely unlikely that no one is changing the ratings of any articles.
